angular-app/firebase.json

```diff
@@ -0,0 +1,28 @@
+{
+  "hosting": [
+    {
+      "target": "ensemble",
+      "public": "dist/ensemble",
+      "ignore": [
+        "**/.*"
+      ],
+      "headers": [
+        {
+          "source": "*.[0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f].+(css|js)",
```

Comment:
I'm assuming this is auto-generated? Seems a little superfluous.

Reply:
Yes, it was autogenerated with the CLI. I'm inclined to not change it until we understand the configuration files a little better.